Eniko/index.js

```diff
@@ -0,0 +1,24 @@
+
+let chrs = 'ABCDEFGH';
+
+// 🚀 Write Program to picking one random character from chrs.
+
+console.log(chrs[Math.floor(Math.random() * 8)]);
+
+// ---------############------------
+
+let sy = '%& § !* @= ';
```

Comment:
Here is the solution of second task.
let sy = '%& § !* @= ';
let syWithNoSpaces = sy.replace(/ /g, "");
console.log(syWithNoSpaces);
let randomSy = syWithNoSpaces[Math.floor(Math.random() * syWithNoSpaces.length)];
console.log("is not empty:", randomSy);